scene/resources/particle_process_material.cpp

```diff
@@ -595,14 +610,14 @@ void ParticleProcessMaterial::_update_shader() {
 	if (emission_curve.is_valid()) {
 		code += "	parameters.color.rgb *= 1.0 + texture(emission_curve, vec2(lifetime)).r;\n";
 	}
-	code += "}\n";
+	code += "}\n\n";
 
 	code += "vec3 calculate_initial_position(inout uint alt_seed) {\n";
 	code += "	float pi = 3.14159;\n";
-	code += "	float degree_to_rad = pi / 180.0;\n";
-	code += "	vec3 pos = vec3(0.);\n";
+	code += "	vec3 pos = vec3(0.0);\n";
+	code += "	{ // Emission shape.\n";
 	if (emission_shape == EMISSION_SHAPE_POINT) {
-		code += "	 pos = vec3(0.);\n";
+		code += "	 	pos = vec3(0.0);\n";
```

Comment:
This is technically redundant with the variable declaration above. But I guess we want the codegen to be explicit.